x/superfluid/keeper/gov/gov_test.go

```diff
@@ -59,6 +59,7 @@ func (suite *KeeperTestSuite) TestHandleSetSuperfluidAssetsProposal() {
 		assets         []types.SuperfluidAsset
 		expectedAssets []types.SuperfluidAsset
 		expectErr      bool
+		expectedEvent  string
 	}
 	testCases := []struct {
```

Comment:
Are we able to / should we break down this actions struct by item? Maybe its not a big deal but just seeing a lump of five items together doesn't make the table driven test super easy to figure out what is getting tested.

Reply:
Good idea, naming fields is preferable for readability IMO